sys/shell/shell.c

```diff
@@ -104,6 +121,10 @@ static shell_command_handler_t find_handler(
         handler = search_commands(_builtin_cmds, command);
     }
 
+    if (handler == NULL) {
+        handler = search_commands_xfa(command);
+    }
```

Comment:
Should `shell_command_xfa` maybe be a pseudo-module, so we are able to switch of this experimental code in production?

Reply:
that sounds useful, will do

Reply:
done